fizz_buzz/fizzBuzzQuiz.js

```diff
@@ -0,0 +1,18 @@
+function fizzBuzz(max){
+	for(i=0; i<=max; i++){
+  	if(i % 3 == 0 && i % 5 == 0){
```

Comment:
I think you're mixing tabs and spaces. You should probably pick one or the other. It's made the indentation inconsistent and the code is harder to read because of the indentation.